src/Component/Drawer/Drawer.jsx

```diff
@@ -1,13 +1,15 @@
 import './Drawer.css'
-export const Drawer = () => {
+import { Link } from 'react-router-dom'
+export const Drawer = ({sideBar}) => {
   return (
-    <>
-      <div className="drawerDiv page">
+     <div className={`drawerDiv page ${sideBar ? "displayBlock" : 'displayNone' }`}>
         <ul className="drawerUl">
+            <Link to='/'>
```

Comment:
Try using the ```NavLink``` component, so that we can use custom styles when the user is on the page.

Reply:
okk i will use it.